src/main/java/org/broadinstitute/barclay/argparser/LegacyCommandLineArgumentParser.java

```diff
@@ -1050,6 +1055,9 @@ private OptionDefinition(final Field field, final String name, final String shor
             this.isCommon = isCommon;
             this.mutuallyExclusive = new HashSet<String>(Arrays.asList(mutuallyExclusive));
             this.printOrder = printOrder;
+            if (this.field.getAnnotation(Hidden.class) != null) {
+                logger.warn("Hidden argument --{} is not handled", this.name);
```

Comment:
I don't think Hidden or Advanced were present or used in Picard, and we don't really want to invest in adding more features to the old parser, so I think what you've done here is exactly the right thing.